webform_confirm_email.install

```diff
@@ -161,6 +161,24 @@ function webform_confirm_email_uninstall() {
 // **************** UPDATES ****************
 // *****************************************
 
+/**
+ * Set created time for emails that didn’t get one in 7206.
```

Comment:
7207, I guess...

Reply:
Nope it’s about 7206. The values are only copied to a new table in 7207, but they are modified in 7206. The change in 7207 should have been in hook_schema though …

Reply:
Never mind, you are right.